data/dasch_ark_registry.ini

```diff
@@ -311,7 +311,7 @@ UsePhp: true
 # WordWeb
 
 [0826]
-Host: 0826-test-server.dasch.swiss
+Host: admin.dasch.swiss
```

Comment:
This should be `app.dasch.swiss`. `admin.dasch.swiss` is deprecated and will be removed at some point.

Reply:
Thanks! I have done that.